.github/actions/docker-run/action.yml

```diff
@@ -86,8 +86,12 @@ runs:
           ${{ inputs.device }}
           -w ${{ github.workspace }}
         run: |
-          if [ ${{ inputs.install_wheel }} ]; then
+          set -eu
+
+          install_wheel=${{ inputs.install_wheel }}
+          if [[ "${install_wheel,,}" == "true" ]]; then
```

Comment:
What is this double comma syntax?

Reply:
It converts to lowercase.  So the comparison is case insensitive.